src/cli/Cli.ts

```diff
@@ -852,6 +860,32 @@ export class Cli {
     return filteredArgs;
   }
 
+  private static loadValuesFromAccessToken(args: { options: minimist.ParsedArgs }) {
+    if (!auth.service.connected) {
+      return;
+    }
+
+    const accessToken = auth.service.accessTokens[auth.defaultResource].accessToken;
+    if (Utils.isAppOnlyAccessToken(accessToken)) {
+      return;
+    }
+
+    const optionNames: string[] = Object.getOwnPropertyNames(args.options);
+    optionNames.forEach(option => {
+      const value = args.options[option];
+      if (!value || typeof value !== 'string') {
+        return;  
+      }
+
+      if (value.toLowerCase() === '@me.id') {
+        args.options[option] = Utils.getUserIdFromAccessToken(accessToken);
```

Comment:
Let's move calling `Utils.getUserIdFromAccessToken` and `Utils.getUserNameFromAccessToken` outside `forEach` so that we call these methods only once. They're pretty expensive and if we get these values once, we should reuse them for subsequent replacements.